locales/locales.go

```diff
@@ -0,0 +1,30 @@
+package locales
+
+import (
+	"github.com/nicksnyder/go-i18n/v2/i18n"
+	"golang.org/x/text/language"
+	"gopkg.in/yaml.v2"
+)
+
+var supportedLanguages = []string{"en", "pt"}
+var bundle *i18n.Bundle
+
+func init() {
+	bundle = i18n.NewBundle(language.Portuguese)
+	bundle.RegisterUnmarshalFunc("yml", yaml.Unmarshal)
+
+	for _, lang := range(supportedLanguages) {
+		bundle.MustLoadMessageFile(lang + ".yml")
+	}
+}
+
+// Receives the desired message, language, and contextual data (such as names and quantities)
+func Translate(id string, language string, metadata map[string]string) (string, error) {
```

Comment:
All comments should start with func name e.g:

` // Translate receives the desired message, language, and contextual data (such as names and quantities)`